### PR TITLE
Remove else

### PR DESCRIPTION
**Pull request by lilianjin**

### CODE REVIEW

**Comment by 36864** [comment]
What does this fix?

File: src/Illuminate/Http/Resources/CollectsResources.php [hunk 1]
```diff
@@ -21,11 +21,9 @@ protected function collectResource($resource)
...
         $collects = $this->collects();
 
-        if ($collects && ! $resource->first() instanceof $collects) {
-            $this->collection = $resource->mapInto($collects);
-        } else {
-            $this->collection = $resource->toBase();
-        }
+        $this->collection = $collects && ! $resource->first() instanceof $collects
+            ? $resource->mapInto($collects)
+            $resource->toBase();
```
Comment (browner12) [inline src/Illuminate/Http/Resources/CollectsResources.php]: missing a `:`

**Comment by phanan** [comment]
You've turned some perfectly clean code into something pretty unreadable.

**Comment by phanan** [comment]
Oh, and syntax error as well…

**Comment by browner12** [comment]
thank you for your contribution, but traditionally these types of PRs are not accepted when they are simply for code style changes.

this is part of an ongoing discussion in the FW

**Comment by tillkruss** [comment]
Code is broken and harder to read than before.

**Pull request closed without merging**